test/user/user_list.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require('supertest');
+const app = require('../../app');
+const { TOKEN_NOT_PROVIDED } = require('../../app/constants/errors').responses;
+const { AUTHENTICATION_ERROR } = require('../../app/errors');
+
+const { generateUsers, credentials } = require('../factory/factory_users');
+
+describe('User list: GET /users', () => {
+  beforeEach(async () => {
+    await generateUsers(10);
```

Comment:
You can put the number of users (10) in a constants file inside the tests folder.